cluster/src/main/java/com/hartwig/pipeline/CommandLineOptions.java

```diff
@@ -64,7 +63,8 @@
     private static final String CMEK_FLAG = "cmek";
     private static final String SHALLOW_FLAG = "shallow";
     private static final String ZONE_FLAG = "zone";
-    private static final String REF_GENOME_VERSION = "ref_genome_version";
+    private static final String REF_GENOME_VERSION_FLAG = "ref_genome_version";
+    private static final String JSON_FLAG = "json";
```

Comment:
Could we come up with a name that is more descriptive of its function rather than just indicative of its file format? Like maybe `sample_json` or something?

Reply:
sample_json sounds reasonable. I'll use that.